products/models.py

```diff
@@ -22,7 +23,9 @@ class Meta:
         db_table = 'images'
 
 class ProductGroup(models.Model):
-    name = models.CharField(max_length=50)
+    name    = models.CharField(max_length=50)
+    img_url = models.URLField(max_length=1000)
```

Comment:
full name으로 만들어주세요-!
```suggestion
    image_url = models.URLField(max_length=1000)
```